shadow-pods/cmd/loki-logger/main.go

```diff
@@ -100,6 +138,29 @@ func (q *Query) Run(start string) (string, error) {
 }
 
 func main() {
+	loki_install_type := os.Getenv("LOKI_INSTALL_TYPE")
+	if loki_install_type == "" {
+		log.Fatal("LOKI_INSTALL_TYPE env variable is not set.")
+	} else if loki_install_type != "openshift" && loki_install_type != "dev" {
+		log.Fatal("LOKI_INSTALL_TYPE has to be either `openshift` or `dev`.")
+	}
+
+	log_type := os.Getenv("LOG_TYPE")
+	tls_cert_file := os.Getenv("TLS_CERT_FILE")
+	tls_key_file := os.Getenv("TLS_KEY_FILE")
```

Comment:
@Seep-Goel  this looks good, could you perhaps add some comment on how to retrieve or generate the required `TLS_CERT_FILE` and `TLS_KEY_FILE' ?

Reply:
Sure, will add comments in the script.